tools/hdf_converter/hdf_converter.xml

```diff
@@ -24,6 +26,12 @@
         </test>
     </tests>
     <help><![CDATA[
-        Tool to convert apLCMS output to csv file formated in accordance to RamClustR input requirements.
+        Tool to convert apLCMS output to csv file formated in accordance to `RamClustR (CSV) <https://umsa.cerit-sc.cz/root?tool_id=testtoolshed.g2.bx.psu.edu/repos/recetox/ramclustr_csv/ramclustr_csv/1.1.0+galaxy0>`_ input requirements.
+
+        **Workflow position**
+
+        - Upstream tool: `apLCMS-unsupervised <https://umsa.cerit-sc.cz/root?tool_id=testtoolshed.g2.bx.psu.edu/repos/recetox/recetox_aplcms_unsupervised/recetox_aplcms_unsupervised/6.6.6+galaxy1>`_ or `apLCMS-Hybrid <https://umsa.cerit-sc.cz/root?tool_id=testtoolshed.g2.bx.psu.edu/repos/recetox/recetox_aplcms_hybrid/recetox_aplcms_hybrid/6.6.6+galaxy1>`_
```

Comment:
These links are going to change as early as we switch to using the Main Tool Shed (and in many other circumstances). I recommend using links to github development repository as the canonical reference.

Reply:
so e.g. https://github.com/RECETOX/galaxytools/tree/master/tools/ramclustr